From: Jeremy Bicha <jbicha@debian.org>
Date: Wed, 16 Jan 2019 21:08:04 -0500
Subject: Restore peripherals display keys

Add dummy keys to ease the transition for
org.gnome.desktop.peripherals.touchscreen display and
org.gnome.desktop.peripherals.tablet display

Current users of the old keys are gnome-settings-daemon,
gnome-control-center, gnome-flashback, mutter and ukwm
---
 schemas/org.gnome.desktop.peripherals.gschema.xml.in | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/schemas/org.gnome.desktop.peripherals.gschema.xml.in b/schemas/org.gnome.desktop.peripherals.gschema.xml.in
index 7ee3882..014d7f7 100644
--- a/schemas/org.gnome.desktop.peripherals.gschema.xml.in
+++ b/schemas/org.gnome.desktop.peripherals.gschema.xml.in
@@ -132,6 +132,11 @@
     </key>
   </schema>
   <schema id="org.gnome.desktop.peripherals.tablet">
+    <key name="display" type="as">
+      <default>["", "", ""]</default>
+      <summary>Deprecated tablet output mapping</summary>
+      <description>This key is deprecated. Use "org.gnome.desktop.peripherals.tablet output" instead</description>
+    </key>
     <key name="output" type="as">
       <default>["", "", ""]</default>
       <summary>Drawing tablet mapping</summary>
@@ -198,6 +203,11 @@
     </key>
   </schema>
   <schema id="org.gnome.desktop.peripherals.touchscreen">
+    <key name="display" type="as">
+      <default>["", "", ""]</default>
+      <summary>Deprecated touchscreen output mapping</summary>
+      <description>This key is deprecated. Use "org.gnome.desktop.peripherals.touchscreen output" instead</description>
+    </key>
     <key name="output" type="as">
       <default>["", "", ""]</default>
       <summary>Touchscreen output mapping</summary>
